go.mod

```diff
@@ -5,7 +5,7 @@ go 1.13
 require (
 	github.com/akutz/gofsutil v0.1.2
 	github.com/container-storage-interface/spec v1.2.0
-	github.com/coreos/etcd v3.3.20+incompatible // indirect
+	github.com/coreos/etcd v3.3.25+incompatible
```

Comment:
We do not use etcd in vSphere CSI driver. This may be added by other Kubernetes Dependency for e2e test cases. 
Considering this I think we do not need this change. We should look into which dependency has added this and bump up that dependency rather than pinning indirect dependency in go.mod file.

I think`go mod tidy` may revert this back. Can you confirm?

Reply:
`go mod tidy` just adds ` // indirect` to the line and cleans up `go.sum`. I've pushed the changes with `go mod tidy`, take a look